Server/package.json

```diff
@@ -46,7 +46,8 @@
     "@fastify/cors": "10.0.1",
     "cross-env": "7.0.3",
     "sqlite3": "5.1.6",
-    "ws": "8.17.1"
+    "ws": "8.17.1",
+    "@aws-sdk/client-s3" : "3.750.0"
```

Comment:
I think you can kick this one out again, since you are doing all the AWS S3 stuff in Base.